### ISSUE TITLE
Add a pattern for an Accessibility Statement

### ISSUE THREAD

**Issue by abbott567** [alphagov/govuk-design-system#1225]
There is a lot of confusion around Accessibility Statements. There is a couple of templates on GOV.UK but nothing mentioned in the design system.

There is a different template for content published on GOV.UK and one for public sector websites published elsewhere. 

It took me quite a bit of time to find and decipher everything I needed. It feels like at the very least the one for publishing on GOV.UK belongs in the Design System. Especially as they're a legal requirement and should be consistent.

**Comment by timpaul** [comment]
Related to this - a user has commented that there doesn’t appear to be any reference to including accessibility statements links within footers, and that guidance on this would be useful (including whether the link should be 'Accessibility' or 'Accessibility statement')

**Comment by StephenGill** [comment]
There is information on this, but it's in the Service Manual: https://www.gov.uk/service-manual/helping-people-to-use-your-service/publishing-information-about-your-services-accessibility

I think there's a similar issue with privacy notices and cookies pages.

Should we create patterns for:

- accessibility statements
- privacy notices
- cookies pages

... each with minimal information and a link to the relevant Service manual content - at least for now, so there's something findable in the design system?
